$NetBSD: patch-bn,v 1.4 2005/12/15 21:28:48 wennmach Exp $

If we have krb4, do not rename the functions, else we're not going
to find them.

--- rxkad/krb4-protos.h.orig	2005-12-09 15:18:27.000000000 +0100
+++ rxkad/krb4-protos.h	2005-12-09 15:20:14.000000000 +0100
@@ -36,6 +36,7 @@
 #ifndef __KRB4_PROTOS_H
 #define __KRB4_PROTOS_H
 
+#ifndef HAVE_KRB4
 #define krb4_get_int _rxkad_krb4_get_int
 #define krb4_put_int _rxkad_krb4_put_int
 #define krb4_get_address _rxkad_krb4_get_address
@@ -48,6 +49,7 @@
 #define krb4_atime_to_life _rxkad_krb4_atime_to_life
 #define krb4_life_to_time _rxkad_krb4_life_to_time
 #define decomp_ticket _rxkad_decomp_ticket
+#endif /* ndef HAVE_KRB4 */
 
 int
 krb4_get_int(void *f, u_int32_t *to, int size, int lsb);
